packages/components/src/DataList/components/DataListAction/DataListAction.tsx

```diff
@@ -34,6 +35,22 @@ export function DataListAction<T extends DataListObject>({
     }
   }
 
+  if (actionUrl) {
+    return (
+      <a
+        href={actionUrl}
+        target="_blank"
```

Comment:
Opening in a new tab without the user's intent is an anti-pattern and if it happens by surprise can make the experience seem broken as it appears in the current tab that nothing is happening. I know in the demo we got from David the case was to allow multiple workflows to be kicked off without revisiting the DataList, but that was with him cmd+clicking.

Reply:
Ahhhh ok! I read "new tab" in the ticket and went with it, but now I understand why we shouldn't do that. Thanks Chris!